AssistantV1.playground/Sources/Helpers.swift

```diff
@@ -16,6 +16,142 @@ public func setupAssistantV1() -> Assistant {
+                    return
+                }
+
+                workspaceID = workspace.workspaceID
+            }
+        }
+    }
+
+    while workspaceID == nil { sleep(1) }
+    return workspaceID!
+}
+
+public func createSampleIntent(workspaceID: String) {
+    let assistant = setupAssistantV1()
+    var created = false
+    assistant.getIntent(workspaceID: workspaceID, intent: "hello") {
+        _, error in
+
+        if let error = error {
+            if case let .http(statusCode, _, _) = error, statusCode == 404 {
+                // Not Found -- fall through to creation
+            } else {
+                print(error.localizedDescription)
+                return
+            }
+
+            assistant.createIntent(
+                workspaceID: workspaceID,
+                intent: "hello",
+                examples: [
+                    CreateExample(text: "Good afternoon"),
+                    CreateExample(text: "Hi there")
+                ])
+            {
+                _, error in
+
+                if let error = error {
+                    print(error.localizedDescription)
+                    return
+                }
+
+                created = true
+            }
+        }
+
+        created = true
+    }
+    while !created { sleep(1) }
+    return
+}
+
+public func deleteSampleIntent(workspaceID: String) {
+    let assistant = setupAssistantV1()
+    assistant.deleteIntent(workspaceID: workspaceID, intent: "hello") {
+        _, error in
+
+        if let error = error {
+            print(error.localizedDescription)
+            return
+        }
+    }
+}
+
+public func createSampleEntity(workspaceID: String) {
+    let assistant = setupAssistantV1()
+    var created = false
+    assistant.getEntity(workspaceID: workspaceID, entity: "beverage") {
+        _, error in
+
+        if let error = error {
+            if case let .http(statusCode, _, _) = error, statusCode == 404 {
+                // Not Found -- fall through to creation
+            } else {
+                print(error.localizedDescription)
+                return
+            }
+
+            assistant.createEntity(
+                workspaceID: workspaceID,
+                entity: "beverage",
+                values: [
+                    CreateValue(value: "water"),
+                    CreateValue(value: "orange juice"),
+                    CreateValue(value: "soda")
+                ])
+            {
+                _, error in
+
+                if let error = error {
+                    print(error.localizedDescription)
+                    return
+                }
+
+                created = true
+            }
+        }
+
+        created = true
+    }
+    while !created { sleep(1) }
+    return
+}
+
+public func deleteSampleEntity(workspaceID: String) {
+    let assistant = setupAssistantV1()
+    assistant.deleteEntity(workspaceID: workspaceID, entity: "beverage") {
```

Comment:
To make this a little more robust, the "beverage" string should be put into a constant to be used in both `createSampleEntity()` and `deleteSampleEntity()`.

Reply:
Will do.